tests/test_playbooks/vars/host.yml

```diff
@@ -5,3 +5,9 @@ host:
   hostgroup: "test_group"
   domain: "example.net"
   subnet: Test subnet4
+  arch: superARCH
```

Comment:
Is this a 42 bit architecture?

Reply:
192!

It's a reference to https://en.wikipedia.org/wiki/SuperH ;-)